w17-0118-intro-to-data-science.md

```diff
@@ -0,0 +1,71 @@
+# Welcome to DSC's Winter 2017 Weeklies!
+
+These notes are from Winter 2017's first weekly event. For more events, make sure you've signed up for notifications from our Facebook group at [Waterloo Data Science Club](https://www.facebook.com/groups/1667117806888083/).
+
+Notes are not comprehensive, come to weekly sessions to hear the full discussions, meet teammates, and more!
+
+# Cornerstones of data science
+
+There's a wealth of tools for data science. Where do you get started? Luckily for you, Jason Liu's narrowed it down to three core areas.
+Here are the things you should master:
+
+##### data transforms:
+SQL, pandas, [dplyr](https://github.com/hadley/dplyr)
+
+- Sarth recommends [SQLZOO](http://sqlzoo.net/) for picking up SQL
+
```

Comment:
add a line for pandas.

i recommend https://www.youtube.com/watch?v=5JnMutdy6Fw as a source to learn it from scratch.